server/src/http_server/pages/blog/md/html.rs

```diff
@@ -19,6 +20,7 @@ use crate::AppConfig;
 pub struct SyntaxHighlightingContext {
     pub(crate) theme: syntect::highlighting::Theme,
     pub(crate) syntax_set: syntect::parsing::SyntaxSet,
+    pub(crate) current_article_path: Option<String>,
 }
 
 impl Default for SyntaxHighlightingContext {
```

Comment:
The `current_article_path` field is used to store the path of the current article being processed